docs/java/guides/increase-log-level.md

```diff
@@ -1,88 +0,0 @@
----
-id: change-log-level
-title: Change log level
-hide_title: false
-hide_table_of_contents: false
-sidebar_label: Increase log level
-description: Change log levels for your Cloud SDK application for efficient debugging
-keywords:
-- sap
-- cloud
-- sdk
-- cloud native
-- cloud sdk
-- sap cloud sdk
-image:
----
-
-## When running app/debugging on localhost ##
-
-For applications that run on localhost you can influence the log level settings via Maven.
-
-### Change Default Log Level
-
-To change the default log level, add this argument to your `mvn` run command:
-```makefile
--Dorg.slf4j.simpleLogger.defaultLogLevel=debug
```

Comment:
This part was actually incorrect, it only works if one is using the `slf4j-simple` implementation. But in our archetypes we ship Logback, so usually that won't be the case. Hence removing it.